client/app/components/Button.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
```

Comment:
Yeah, will refactor this login screen with native-base components I think - just needed to get my hands dirty with writing some RN code a little first :D